projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-transient.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * REST API endpoint for editing Jetpack Transients.
+ *
+ * @package automattic/jetpack
+ * @since 9.7.0
+ */
+
+/**
+ * Edit and delete transients.
+ *
+ * @since 9.7.0
+ */
+class WPCOM_REST_API_V2_Endpoint_Transient extends WP_REST_Controller {
+	/**
+	 * Constructor.
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'transients';
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Called automatically on `rest_api_init()`.
+	 *
+	 * /sites/$site/transients/$name/delete
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/(?P<name>\w{1,172})/delete/',
```

Comment:
I was wrong about the verb here. Let's remove it and change the endpoint method.

```suggestion
			'/' . $this->rest_base . '/(?P<name>\w{1,172})',
```